### PR TITLE
Docs: Access Monitoring

### PR DESCRIPTION
**Pull request by smallinsky**
### What 

Add Access Monitoring docs providing  Access Monitoring SQL Query examples.

Access Monitoring is default enable for all Cloud products.

### CODE REVIEW

File: docs/config.json [hunk 1]
```diff
@@ -494,6 +494,14 @@
             }
           ]
         },
+        {
+          "title": "Access Monitoring",
+          "slug": "/access-controls/access-monitoring/",
+          "forScopes": [
+            "team",
+            "cloud"
+          ]
```
Comment (r0mant) [inline docs/config.json:503]: We should also add a guide explaining how to set it up for self-hosted clusters. We can do it as a follow-up to this PR though so we can deploy these docs quicker.

File: docs/pages/access-controls/access-monitoring.mdx [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+This guide will help you understand how to use Access Monitoring to monitor access to resources in your Teleport cluster.
+
+
+## Prerequisites
+- Teleport Cloud cluster v14.2.2 or later.
+
+## Access Monitoring RBAC Permissions
+
+To access the Access Monitoring interface, a user must have a role that allows
+`list`, `read` and `use` verbs on the `audit_query` and `audit_query` resources.
```
Comment (r0mant) [inline docs/pages/access-controls/access-monitoring.mdx:20]: Typo?
```suggestion
`list`, `read` and `use` verbs on the `security_report` and `audit_query` resources.
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+      verbs:
+      - list
+      - read
+      - use
+```
+
+## Query Editor
+The Query Editor in Teleport Access Monitoring provides users with a powerful interface to interactively query audit logs and generate insightful reports.
+It offers a user-friendly environment for constructing SQL queries tailored to specific use cases,
+allowing users to gain deep insights into access patterns and security events within Teleport cluster.
```
Comment (r0mant) [inline docs/pages/access-controls/access-monitoring.mdx:46]:
```suggestion
allowing users to gain insights into access patterns and security events within Teleport cluster.
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+
+Within the Query Editor, users have access to a diverse range of SQL views representing various audit events captured by Teleport.
+These views provide a comprehensive overview of activities within the Teleport cluster, enabling users to construct precise queries tailored to their monitoring needs.
```
Comment (r0mant) [inline docs/pages/access-controls/access-monitoring.mdx:49]: I would rephrase this sentence to: "Users can write custom SQL queries for these views to build custom reports akin to querying a relational database."

File: docs/pages/access-controls/access-monitoring.mdx [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+      - read
+      - use
+```
+
+## Query Editor
+The Query Editor in Teleport Access Monitoring provides users with a powerful interface to interactively query audit logs and generate insightful reports.
+It offers a user-friendly environment for constructing SQL queries tailored to specific use cases,
+allowing users to gain deep insights into access patterns and security events within Teleport cluster.
+
+Within the Query Editor, users have access to a diverse range of SQL views representing various audit events captured by Teleport.
```
Comment (r0mant) [inline docs/pages/access-controls/access-monitoring.mdx:48]: Nit: Fewer "fluff" words.
```suggestion
Within the Query Editor, users have access to a number of SQL views representing audit events captured by Teleport.
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 6]
```diff
@@ -0,0 +1,208 @@
...
+user_login
+user_password_change
+windows_desktop_session_end
+windows_desktop_session_start
+```
+
+To access the Query Editor, navigate to the `Access Monitoring` section in the Teleport UI and click on the `Query Editor` tab.
+
+
+### Here are a few query examples that demonstrate the capabilities of Access Monitoring
```
Comment (r0mant) [inline docs/pages/access-controls/access-monitoring.mdx:93]:
```suggestion
### SQL query examples
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 7]
```diff
@@ -0,0 +1,208 @@
...
+GROUP BY
+  event_date,
+  identity_user
+ORDER BY
+  event_date) AS data WHERE unique_ip_count > 1;
+```
+
+![cert ip count](../../img/access-monitoring/cert_create_ip_count2.png)
+
+- Sow all the unique IP addresses that were used by the user 'admin-annie':
```
Comment (r0mant) [inline docs/pages/access-controls/access-monitoring.mdx:142]:
```suggestion
- Show all the unique IP addresses that were used by the user 'admin-annie':
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 8]
```diff
@@ -0,0 +1,208 @@
...
+WHERE
+  request.id = review.id
+```
+![access request](../../img/access-monitoring/access_request.png)
+
+
+### Query Examples with Assist AI
+
+AI assisted query is a feature of the Teleport Access Monitoring Query editor.
+It provide SQL Query Suggestion suggestions based on the audit query schema available in your Teleport cluster.
```
Comment (r0mant) [inline docs/pages/access-controls/access-monitoring.mdx:180]:
```suggestion
AI assisted query feature provides SQL query suggestions based on the audit query schema available in your Teleport cluster.
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 9]
```diff
@@ -0,0 +1,208 @@
...
+
+
+![assist query](../../img/access-monitoring/assist_query_1.png)
+
+
+In the result the Assist based on human text will generate the SQL query:
```
Comment (r0mant) [inline docs/pages/access-controls/access-monitoring.mdx:186]:
```suggestion
Teleport Assist will generate the SQL query based on the user prompt message:
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 10]
```diff
@@ -0,0 +1,208 @@
...
+![assist query result](../../img/access-monitoring/assist_query_2.png)
+
+# Access Monitoring Reports
+Access Monitoring provides pre built reports to help you monitor and analyze access patterns in your Teleport cluster.
```
Comment (r0mant) [inline docs/pages/access-controls/access-monitoring.mdx:190]:
```suggestion
Access Monitoring provides a number of pre-built reports.
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 11]
```diff
@@ -0,0 +1,208 @@
...
+
+- Privileged Access Report
```
Comment (r0mant) [inline docs/pages/access-controls/access-monitoring.mdx:192]: I would make this (and future reports we add) into a separate sub-section.
```suggestion
## Privileged Access Report
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 12]
```diff
@@ -0,0 +1,208 @@
...
+
+    - Database sessions with weak security
+    - SSH sessions with weak security
+    - Kubernetes API calls with weak security
+    - Privileged Postgres sessions
+    - Kube Execs
+    - Long lived certificates
+    - Long-lived join tokens
+    - Root SSH sessions
+    - System Kubernetes API calls
```
Comment (r0mant) [inline docs/pages/access-controls/access-monitoring.mdx:204]: Can we expand each of these to:

1. Give brief explanation of what it shows.
2. Show the underlying SQL query it runs.
3. Provide recommendation similar to what we show in the UI.

I would turn each of these into a sub-section as well e.g.

```
### Database sessions with weak security

... explanations, SQL, recommendation

### SSH sessions with weak security

...
```

Screenshots for each individual section aren't necessary here IMO, just a single screenshot you have below should be fine.

**Review by r0mant — COMMENTED** [review]
Left some suggestions.

File: docs/img/access-monitoring/access_request_details.png
Comment (zmb3) [inline docs/img/access-monitoring/access_request_details.png:1]: The line wrapping here, especially for the `reason` column looks a bit awkward and unprofessional. Instead of `SELECT *`, can we select just a few fields that will render nicely to make for a better screenshot?

File: docs/pages/access-controls/access-monitoring.mdx [hunk 13]
```diff
@@ -0,0 +1,208 @@
+---
+title: Getting Started with Access Monitoring
+description: Learn how to use Access Monitoring.
+---
+
+
```
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:6]:
```suggestion
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 14]
```diff
@@ -0,0 +1,208 @@
...
+
+Teleport Access Monitoring allows to query and analyze the audit events generated by Teleport.
```
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:8]:
```suggestion
Teleport Access Monitoring allows you to query and analyze the audit events generated by Teleport.
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 15]
```diff
@@ -0,0 +1,208 @@
...
+It provides a SQL interface to query the audit logs and generate reports.
+Access Monitoring is a powerful tool to understand and analyze the access patterns in your Teleport cluster.
```
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:10]: The first 3 sentences of this paragraph all say basically the same thing: access monitoring allows you to query and analyze audit logs.

Can we remove some of the duplication here and simplify the language?

File: docs/pages/access-controls/access-monitoring.mdx [hunk 16]
```diff
@@ -0,0 +1,208 @@
...
+This guide will help you understand how to use Access Monitoring to monitor access to resources in your Teleport cluster.
+
+
+## Prerequisites
+- Teleport Cloud cluster v14.2.2 or later.
```
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:15]: If this guide is only for cloud, I wonder if we even need to mention the specific version. Maybe we just say "running Teleport 14 or later" since we know we aren't running any v14 tenants on versions this old.

File: docs/pages/access-controls/access-monitoring.mdx [hunk 17]
```diff
@@ -0,0 +1,208 @@
...
+
+## Access Monitoring RBAC Permissions
+
+To access the Access Monitoring interface, a user must have a role that allows
+`list`, `read` and `use` verbs on the `audit_query` and `audit_query` resources.
+By default the built-in `auditor` role has these permissions or you can create a custom role with these permissions e.g:
```
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:21]:
```suggestion
The preset `auditor` role has these permissions by default. Alternatively, you can create a custom role with these permissions:
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 18]
```diff
@@ -0,0 +1,208 @@
...
+```yaml
+kind: role
```
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:23]: Missing `version`?

File: docs/pages/access-controls/access-monitoring.mdx [hunk 19]
```diff
@@ -0,0 +1,208 @@
...
+      verbs:
+      - list
+      - read
+      - use
+    - resources:
+      - audit_query
+      verbs:
+      - list
+      - read
+      - use
```
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:40]:
```suggestion
    - resources:
      - audit_query
      - security_report
      verbs:
      - list
      - read
      - use
```

File: docs/pages/access-controls/access-monitoring.mdx (hunk 4 above)
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:49]: This feels duplicative too.

We've already mentioned  a _powerful interface_, a _diverse range of views_, and _deep insights_. It feels like we're just repeating the same sales pitch with different words.

File: docs/pages/access-controls/access-monitoring.mdx [hunk 20]
```diff
@@ -0,0 +1,208 @@
...
+FROM
+  cert_create
+GROUP BY
+  event_date,
+  identity_user
+ORDER BY
+  event_date;
+```
+
+![cert ip](../../img/access-monitoring/cert_create_ip.png)
```
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:122]: I think we probably only need to show one screenshot. We can list multiple example queries in the docs, but all of these screenshots are basically the same (a query with a table showing the results)

File: docs/pages/access-controls/access-monitoring.mdx [hunk 21]
```diff
@@ -0,0 +1,208 @@
...
+  access_request_create as request, access_request_review as review
+WHERE
+  request.id = review.id
+```
+![access request](../../img/access-monitoring/access_request.png)
+
+
+### Query Examples with Assist AI
+
+AI assisted query is a feature of the Teleport Access Monitoring Query editor.
```
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:179]: The query editor also integrates with Teleport Assist, making it easy to generate queries with a human-readable description of the data you want to retrieve.

File: docs/pages/access-controls/access-monitoring.mdx (hunk 10 above)
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:190]:
```suggestion
Access Monitoring includes several built-in reports to help you monitor and analyze access patterns in your Teleport cluster.
```

File: docs/pages/access-controls/access-monitoring.mdx (hunk 18 above)
Comment (smallinsky) [reply docs/pages/access-controls/access-monitoring.mdx:23]: The version seems to be not necessary. Teleport Will set the default  current version during creation.

File: docs/pages/access-controls/access-monitoring.mdx (hunk 15 above)
Comment (smallinsky) [reply docs/pages/access-controls/access-monitoring.mdx:6]: squeezed to: 

> Access Monitoring is a powerful tool to understand and analyze the access patterns in your Teleport cluster.
This guide will help you understand how to use Access Monitoring to monitor access to resources in your Teleport cluster.

File: docs/img/access-monitoring/access_request_details.png
Comment (zmb3) [inline docs/img/access-monitoring/access_request_details.png:1]: This can now be removed, right?

File: docs/img/access-monitoring/access_request.png
Comment (zmb3) [inline docs/img/access-monitoring/access_request.png:1]: Is this used anymore?

File: docs/img/access-monitoring/cert_create_ip_count2.png
Comment (zmb3) [inline docs/img/access-monitoring/cert_create_ip_count2.png:1]: This can be removed.

File: docs/img/access-monitoring/ip_user_address.png
Comment (zmb3) [inline docs/img/access-monitoring/ip_user_address.png:1]: Remove?

File: docs/pages/access-controls/access-monitoring.mdx [hunk 22]
```diff
@@ -0,0 +1,427 @@
...
+
+To enable Access Monitoring you need to update the Teleport Auth Service configuration:
+```yaml
+auth_service:
+  access_monitoring:
+    enabled: true
+    # AWS role ARN that Teleport will assume to execute Athena SQL queries.
+    role_arn: arn:aws:iam::111111111111:role/access-monitoring-role
+    # S3 bucket where Access Monitoring reports will be stored.
+    report_results: s3://audit-long-example/report_results
```
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:27]: What is `audit-long`? Did you mean to say `audit-log`?

(Do we want people to use the same bucket they use for audit logging or a different one?)

File: docs/pages/access-controls/access-monitoring.mdx [hunk 23]
```diff
@@ -0,0 +1,427 @@
...
+      verbs:
+      - list
+      - read
+      - use
+```
+
+## Query Editor
+
+The Query Editor in Teleport Access Monitoring provides users with a powerful interface to interactively query audit logs and generate insightful reports.
+It offers a user-friendly environment for constructing SQL queries tailored to specific use cases,
```
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:63]:
```suggestion
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 24]
```diff
@@ -0,0 +1,427 @@
...
+The query editor also integrates with Teleport Assist, making it easy to generate queries with a human-readable description of the data you want to retrieve.
+
+![assist query](../../img/access-monitoring/assist_query_1.png)
+
+
+Teleport Assist will generate the SQL query based on the user prompt message:
+![assist query result](../../img/access-monitoring/assist_query_2.png)
+
+# Access Monitoring Reports
+Access Monitoring provides a number of pre-built reports:
```
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:195]:
```suggestion

Access Monitoring provides a number of pre-built reports:
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 25]
```diff
@@ -0,0 +1,427 @@
...
+
+## Privileged Access Report
+    The `Privileged Access Report` reports offers valuable insight into identifying weak security events across infrastructure.
```
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:198]:
```suggestion

The `Privileged Access Report` reports offers valuable insight into identifying weak security events across infrastructure.
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 26]
```diff
@@ -0,0 +1,427 @@
...
+    The report allows to identify the following weak security events:
```
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:199]:
```suggestion
The report allows to identify the following weak security events:
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 27]
```diff
@@ -0,0 +1,427 @@
...
+```
+
+![privileged access report](../../img/access-monitoring/privileged_access_report.png)
+
+
+**Suggestion:**  Setup access requests, device trust and per-session MFA.
+
+### SSH sessions with weak security
+
+Following query identifies SSH sessions with weak security,
```
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:235]:
```suggestion
The following query identifies SSH sessions with weak security,
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 28]
```diff
@@ -0,0 +1,427 @@
...
+# Access Monitoring Reports
+Access Monitoring provides a number of pre-built reports:
+
+## Privileged Access Report
+    The `Privileged Access Report` reports offers valuable insight into identifying weak security events across infrastructure.
+    The report allows to identify the following weak security events:
+
+### Database sessions with weak security
+
+Following query identifies database sessions with weak security
```
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:203]:
```suggestion
The following query identifies database sessions with weak security
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 29]
```diff
@@ -0,0 +1,427 @@
...
+    trusted_device_device_id IS NULL
+GROUP BY
+    event_date,
+    proto,
+    user
+ORDER BY
+    event_date
+```
+
+**Suggestion:**  Setup access requests, device trust and per-session MFA.
```
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:263]:
```suggestion
**Suggestion:**  Set up access requests, device trust and per-session MFA.
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 30]
```diff
@@ -0,0 +1,427 @@
...
+AND
+    trusted_device_device_id IS NULL
+GROUP BY
+    event_date,
+    user
+ORDER BY
+    event_date
+```
+
+**Suggestion:**  Setup access requests, device trust and per-session MFA.
```
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:292]:
```suggestion
**Suggestion:**  Set up access requests, device trust and per-session MFA.
```

File: docs/pages/access-controls/access-monitoring.mdx (hunk 22 above)
Comment (smallinsky) [reply docs/pages/access-controls/access-monitoring.mdx:27]: Yes, It was intended to be audt-log-example thanks for catching this.

File: docs/img/access-monitoring/access_request.png
Comment (smallinsky) [reply docs/img/access-monitoring/access_request.png:1]: no, removed

File: docs/img/access-monitoring/exec_passwd.png
Comment (zmb3) [inline docs/img/access-monitoring/exec_passwd.png:1]: It doesn't look like this is used anymore either.

File: docs/pages/access-controls/access-monitoring.mdx [hunk 31]
```diff
@@ -0,0 +1,428 @@
...
+
+![assist query](../../img/access-monitoring/assist_query_1.png)
+
+
+Teleport Assist will generate the SQL query based on the user prompt message:
+![assist query result](../../img/access-monitoring/assist_query_2.png)
+
+# Access Monitoring Reports
+
+Access Monitoring provides a number of pre-built reports:
```
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:195]:
```suggestion
Access Monitoring provides a number of pre-built reports.
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 32]
```diff
@@ -0,0 +1,428 @@
...
+    user
+ORDER BY
+    event_date
+```
+
+**Suggestion:** Downgrade database connections to less privileged database user.
+
+### Kube Execs
+
+The Following query identifies Kubernetes exec usage.
```
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:319]:
```suggestion
The following query identifies Kubernetes exec usage.
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 33]
```diff
@@ -0,0 +1,428 @@
...
+    user
+ORDER BY
+    event_date
+```
+
+**Suggestion:**  Set up access requests, device trust and per-session MFA.
+
+### Privileged Postgres sessions
+
+The Following query identifies privileged PostgreSQL sessions initiated by the 'postgres' user.
```
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:297]:
```suggestion
The following query identifies privileged PostgreSQL sessions initiated by the 'postgres' user.
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 34]
```diff
@@ -0,0 +1,428 @@
...
+    user
+ORDER BY
+    event_date
+```
+
+**Suggestion:**  Set up access requests, device trust and per-session MFA.
+
+### Kubernetes API calls with weak security
+
+The Following query identifies Kubernetes sessions with weak security,
```
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:268]:
```suggestion
The following query identifies Kubernetes sessions with weak security,
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 35]
```diff
@@ -0,0 +1,428 @@
...
+```
+
+![privileged access report](../../img/access-monitoring/privileged_access_report.png)
+
+
+**Suggestion:**  Set up access requests, device trust and per-session MFA.
+
+### SSH sessions with weak security
+
+The Following query identifies SSH sessions with weak security,
```
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:236]:
```suggestion
The following query identifies SSH sessions with weak security,
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 36]
```diff
@@ -0,0 +1,428 @@
...
+Access Monitoring provides a number of pre-built reports:
+
+## Privileged Access Report
+
+The `Privileged Access Report` reports offers valuable insight into identifying weak security events across infrastructure.
+The report allows to identify the following weak security events:
+
+### Database sessions with weak security
+
+The Following query identifies database sessions with weak security
```
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:204]:
```suggestion
The following query identifies database sessions with weak security
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 37]
```diff
@@ -0,0 +1,428 @@
...
+    FROM_ISO8601_TIMESTAMP(token_expires) - FROM_ISO8601_TIMESTAMP(time) > INTERVAL '1' DAY
+GROUP BY
+    event_date,
+    node_name,
+    host_id
+ORDER BY
+    event_date
+```
+
+**Suggestion:**  Use short-lived tokens to reduce risk of compromise or AWS/GCP/Azure joining when possible.
```
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:385]:
```suggestion
**Suggestion:**  Use short-lived tokens to reduce risk of compromise or delegated joining when possible.
```

Bonus points for linking to Hugo's new joining page.

File: docs/pages/access-controls/access-monitoring.mdx [hunk 38]
```diff
@@ -0,0 +1,428 @@
...
+
+### Root SSH sessions
+
+The Following query identifies SSH sessions initiated by the 'root' user.
```
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:389]:
```suggestion
The following query identifies SSH sessions initiated by the 'root' user.
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 39]
```diff
@@ -0,0 +1,428 @@
...
+    event_date,
+    user
+ORDER BY
+    event_date
+```
+**Suggestion:**  Don’t use `root` for SSH sessions, downgrade access to users with sudo privileges instead.
+
+### System Kubernetes API calls
+
+The Following query identifies Kubernetes API calls initiated by users in the 'system:master' group.
```
Comment (zmb3) [inline docs/pages/access-controls/access-monitoring.mdx:410]:
```suggestion
The following query identifies Kubernetes API calls initiated by users in the 'system:master' group.
```

File: docs/img/access-monitoring/exec_passwd.png
Comment (smallinsky) [reply docs/img/access-monitoring/exec_passwd.png:1]: It used still used in:  https://github.com/gravitational/teleport/pull/38255/files#diff-805c6c2ddbc76ef825e1a87a1d6cb33ca9347b818e5743f5761733ececc7010eR120

File: docs/pages/access-controls/access-monitoring.mdx (hunk 37 above)
Comment (smallinsky) [reply docs/pages/access-controls/access-monitoring.mdx:385]: Done, Thanks for suggestion.

**Review by zmb3 — APPROVED** [review]

File: docs/pages/access-controls/access-monitoring.mdx [hunk 40]
```diff
@@ -0,0 +1,428 @@
+---
+title: Getting Started with Access Monitoring
+description: Learn how to use Access Monitoring.
+---
+
+Access Monitoring is a powerful tool to understand and analyze the access patterns in your Teleport cluster.
```
Comment (r0mant) [inline docs/pages/access-controls/access-monitoring.mdx:6]: Nit: Fewer fluff words.

```suggestion
Access Monitoring allows users to understand and analyze the access patterns in a Teleport cluster.
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 41]
```diff
@@ -0,0 +1,428 @@
...
+This guide will help you understand how to use Access Monitoring to monitor access to resources in your Teleport cluster.
```
Comment (r0mant) [inline docs/pages/access-controls/access-monitoring.mdx:7]: I would remove this sentence because it basically repeats the previous one.

Instead, I think what's missing is a short description of what access monitoring actually is and what it allows you to do. We're jumping straight into talking about configuration without explaining _what_ it actually does and _how_ do you actually monitor and analyze access in your cluster. I think we should give a brief description to help users build a mental model of what it is and how it works.

Something like this:

```
When Access Monitoring is enabled in a cluster, Teleport runs a background service that collects
signals/data from different input sources, builds graphical reports highlighting different usage patterns
within the cluster and allows users to create automated response actions.

The primary signal source for Access Monitoring system currently is the Teleport audit log. Out of the box
Teleport includes a report that searches the audit log for events indicating risky usage patterns (such as SSH
or database sessions without MFA) and provides users with recommendations on suggested actions.

Users are able to write their own custom access monitoring queries by querying the audit log represented as
a number of virtual database views using SQL.
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 42]
```diff
@@ -0,0 +1,428 @@
...
+      - audit_query
+      verbs:
+      - list
+      - read
+      - use
+```
+
+## Query Editor
+
+The Query Editor in Teleport Access Monitoring provides users with a powerful interface to interactively query audit logs and generate insightful reports.
```
Comment (r0mant) [inline docs/pages/access-controls/access-monitoring.mdx:62]: Nit: Fewer fluff words.
```suggestion
The Query Editor in Teleport Access Monitoring provides users with an interface to interactively query audit logs and generate reports.
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 43]
```diff
@@ -0,0 +1,428 @@
...
+Users can write custom SQL queries for these views to build custom reports akin to querying a relational database.
+
+Within the Query Editor, users have access to a number of SQL views representing audit events captured by Teleport.
+Below is a list of the available SQL views:
+
+```
```
Comment (r0mant) [inline docs/pages/access-controls/access-monitoring.mdx:68]: Can this be made a collapsible block?

**Comment by r0mant** [comment]
@smallinsky Can you address the comments here (and add information about how to use `tctl` to export query results we discussed offline), so we can merge and deploy this?

File: docs/pages/access-controls/access-monitoring.mdx [hunk 44]
```diff
@@ -0,0 +1,462 @@
...
+
+The primary data source for Access Monitoring is the Teleport audit log. Out of the box,
+Teleport includes a report that searches the audit log for events indicating risky usage patterns (such as SSH
+or database sessions without MFA) and provides users with recommendations on suggested actions.
+
+Users are able to write their own custom access monitoring queries by querying the audit log.
+
+## Prerequisites
+- Teleport v14 or later.
+- For Self-Hosted Teleport the [AWS Athena Backend](../reference/backends.mdx#athena) is required.
```
Comment (r0mant) [inline docs/pages/access-controls/access-monitoring.mdx:21]:
```suggestion
- For self-hosted Teleport the [AWS Athena Backend](../reference/backends.mdx#athena) is required.
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 45]
```diff
@@ -0,0 +1,462 @@
+---
+title: Getting Started with Access Monitoring
+description: Learn how to use Access Monitoring.
+---
+
+Access Monitoring allows users to understand and analyze the access patterns in a Teleport cluster.
+
```
Comment (r0mant) [inline docs/pages/access-controls/access-monitoring.mdx:7]:
```suggestion
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 46]
```diff
@@ -0,0 +1,462 @@
...
+  access_monitoring:
+    enabled: true
+    # AWS role ARN that Teleport will assume to execute Athena SQL queries.
+    role_arn: arn:aws:iam::111111111111:role/access-monitoring-role
+    # S3 bucket where Access Monitoring reports will be stored.
+    report_results: s3://audit-log-example/report_results
+```
+
+</TabItem>
+<TabItem scope={["cloud","team"]} label="Cloud-Hosted">
```
Comment (r0mant) [inline docs/pages/access-controls/access-monitoring.mdx:41]:
```suggestion
<TabItem scope={["cloud","team"]} label="Teleport Cloud">
```

File: docs/pages/access-controls/access-monitoring.mdx [hunk 47]
```diff
@@ -0,0 +1,462 @@
...
+
+To enable Access Monitoring you need to update the Teleport Auth Service configuration:
+```yaml
+auth_service:
+  access_monitoring:
+    enabled: true
+    # AWS role ARN that Teleport will assume to execute Athena SQL queries.
+    role_arn: arn:aws:iam::111111111111:role/access-monitoring-role
+    # S3 bucket where Access Monitoring reports will be stored.
+    report_results: s3://audit-log-example/report_results
```
Comment (r0mant) [inline docs/pages/access-controls/access-monitoring.mdx:37]: Something that just occurred to me, we should also tell users what IAM permissions Teleport needs to use access monitoring, right? We can add it as a follow up.

File: docs/pages/access-controls/access-monitoring.mdx (hunk 43 above)
Comment (r0mant) [reply docs/pages/access-controls/access-monitoring.mdx:78]: We do support it. You can use "details" block with the "closed" tag:

```
<Details title="Available SQL views" closed>
...
</Details>
```

**Review by r0mant — APPROVED** [review]
Looks mostly good to me now but can you please address a couple remaining comments I left before merging.

File: docs/pages/access-controls/access-monitoring.mdx (hunk 47 above)
Comment (smallinsky) [reply docs/pages/access-controls/access-monitoring.mdx:37]: Right, I will do this as fallow up.

File: docs/pages/access-controls/access-monitoring.mdx (hunk 43 above)
Comment (smallinsky) [reply docs/pages/access-controls/access-monitoring.mdx:78]: Thanks.
